Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -538,6 +538,19 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
           }
       }
 
+    let fixErroredPledgeLink = projectLink
+      .filter { _, subpage, _, _ in subpage == .pledge(.manage) }
+      .map { project, _, vcs, _ -> [UIViewController] in
+        let vc = ManagePledgeViewController.instantiate()
+        vc.configureWith(project: project)
```

Comment:
Should we do some sort of check in here that the `Project` indeed contains a `Backing` for this user? Maybe if the user is logged out of the app at the time they hit the deeplink, for example.

Reply:
Ah yeah good point, I can add that.  This would have the effect of nothing happening after tapping a push/email link when logged out though, wondering if a better way to handle this is either to prompt login (probably tricky to handle the navigation after login), or have some error messaging on `ManagePledgeViewController` 🤔

Reply:
Apologies for the delay - got swept up in that layout bug. I'm going to send the user to the login flow in this case.